src/components/dropdown.tsx

```diff
@@ -81,19 +81,20 @@ const ItemsContainer = styled.div`
 `;
 
 interface DropdownProps {
+  label: string;
   name: string;
   items: string[];
   value: string;
   onClick: Function;
 }
 
-const Dropdown = ({ name, items, onClick, value }: DropdownProps) => {
+const Dropdown = ({ name, items, onClick, value, label }: DropdownProps) => {
   const [open, setOpen] = useState(false);
 
   return (
     <DropdownContainer>
       <LabelContainer onClick={() => setOpen(!open)}>
-        <p>{value || name}</p>
+        <p>{value || label}</p>
```

Comment:
When is this ever used? `value` is never null, according to the types you've assigned here.

Reply:
at first value is an empty string